api/utils/api.js

```diff
@@ -38,6 +38,10 @@ module.exports = {
       collection.gameCount = parseInt(input.gameCount);
     }
 
+    if ("gameIDs" in input) {
+      collection.gameIDs = input.gameIDs ? input.gameIDs.map(e => parseInt(e)) : [];
```

Comment:
We already do the check that `"gameIDs" in input`, right? Maybe you can just take out the if statement and everything else would still work.

Reply:
The reason is that this is actually handling three different response scenarios:
1. Just the collection info only with no info about games (i.e., no `gameIDs` field)
2. An array of game IDs when there are games in the collection
3. An empty array for when there are no games in the collection (i.e., converting `null` to `[]`)

Reply:
Oh I see now. Then would it be a better check that input.gameIDs.length > 0? Just makes it more explicit that we're checking for an empty array vs the presence of the key.

i.e.

```
collection.gameIDs = input.gameIDs.length > 0 ? input.gameIDs.map(e => parseInt(e)) : []; 
```

Reply:
I'm using `array_agg()` in the database query, and it returns `null` instead of an empty array if there are no games. I could change the conditional to `input.gameIDs !== null` if you think that would be more clear?

Reply:
Nah that's fine!